kotlin.web.demo.server/examples/Examples/Canvas/Creatures/Creatures.kt

```diff
@@ -208,13 +208,17 @@ class CanvasState(val canvas: HTMLCanvasElement) {
         get() = v(width.toDouble(), height.toDouble())
     val context = creatures.context
     var valid = false
-    var shapes = ArrayList<Shape>()
+    var shapes = mutableListOf<Shape>()
     var selection: Shape? = null
     var dragOff = Vector()
     val interval = 1000 / 30
 
     init {
-        jq(canvas).mousedown {
+        fun fix(f: Element.(MouseEvent) -> Unit) : Element.(MouseEvent) -> Unit {
```

Comment:
What's this about and why is this needed? Can we avoid showing such ugly stuff in samples?